src/playground/app.ts

```diff
@@ -0,0 +1,53 @@
+import express from 'express';
+import path from 'path';
+import expressHandlebars from 'express-handlebars';
+
+import dotenv from 'dotenv';
+import fs from 'fs';
+
+if (fs.existsSync('.env')) {
+  console.log('Using .env file to supply config environment variables');
+  dotenv.config({ path: '.env' });
+} else {
+  console.log('Using .env.example file to supply config environment variables');
+  dotenv.config({ path: '.env.example' }); // you can delete this after you create your own .env file!
+}
+
+const app = express();
+
+app.use(
+  express.static(path.join(__dirname, 'public'), { maxAge: 31557600000 })
```

Comment:
`Infinity` will do as good as 1000 years :D

How does the caching work if someone updates assets in `/public` for development?

Reply:
It will cache until, well, max-age I think. 
This is why 0 seems a better value.